docker-compose.yml

```diff
@@ -32,5 +37,10 @@ services:
       - es_data:/usr/share/elasticsearch/data
     ports:
       - '43431:9200'
-    networks:
-      - es_net
+
+  redis:
+    image: redis:4-alpine
+    user: redis
+    command: '--save "" --appendonly no'
```

Comment:
You may want to change this so it actually persists to disk. It'd be a shame that you lose all the data because you decided to restart the service.